Don't use show SQL in the tests

stuartwdouglas opened the pull request:
This makes the test logs really large, and most of the time makes things
harder to dubug rather than easier.

Sanne commented on extensions/panache/hibernate-orm-panache/deployment/src/test/resources/application-commented-out.properties:
```diff
@@ -3,5 +3,5 @@ quarkus.datasource.devservices.enabled=false
 #
 #quarkus.hibernate-orm.dialect=org.hibernate.dialect.H2Dialect
-#quarkus.hibernate-orm.log.sql=true
+##quarkus.hibernate-orm.log.sql=true
```
There are several  cases of these `##` double comments.

Sanne commented on extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java:
```diff
@@ -346,33 +346,20 @@ public void handle(Void e) {
-                        hotReplacementHandler.handle(event);
-                    }
-                });
+                httpRouteRouter.route().order(Integer.MIN_VALUE).handler(new TCCLHotReplacementHandler(currentCl));
```
This doesn't seem to belong in this PR?

Sanne reviewed (changes requested):
I agree in principle, but need to remove at least the changes to `VertxHttpRecorder`

stuartwdouglas committed:
Don't use show SQL in the tests

This makes the test logs really large, and most of the time makes things
harder to dubug rather than easier.

Sanne reviewed (approved):
Thanks!

Sanne commented:
Just one failure, and related to `metaspace`. I guess we know it's not related.

thanks!
